usi/src/main/java/daq/usi/BaseSwitchController.java

```diff
@@ -0,0 +1,187 @@
+package daq.usi;
+
+/*
+ * Licensed to Google under one or more contributor license agreements.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+public abstract class BaseSwitchController implements Runnable, SwitchController {
+  /**
+   * Terminal Prompt ends with '#' when enabled, '>' when not enabled.
+   */
+  public static final String CONSOLE_PROMPT_ENDING_ENABLED = "#";
```

Comment:
How can this be part of the base switch -- it seems very implementation specific.

Reply:
That’s actually pretty standard from what I’ve seen which was why I put that in the base class to be used by containsPrompt